src/MICore/Debugger.cs

```diff
@@ -1154,6 +1163,27 @@ private void HandleThreadGroupStarted(Results results)
             }
         }
 
+        public int InferiorByPid(int pid)
+        {
+            foreach (var grp in _debuggeePids)
+            {
+                if ( grp.Value == pid)
+                {
+                    // Inferior names are of the form "iX" where X in the inferior number
+                    string name = grp.Key;
+                    if (name[0] == 'i')
+                    {
+                        int id;
+                        if (Int32.TryParse(name.Substring(1), out id))
```

Comment:
Is signed int the correct type here? I've fixed a few signed vs unsigned ints in the miengine recently.